recipes/r-smartsnp/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-smartsnp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/smartsnp_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/smartsnp/smartsnp_{{ version }}.tar.gz
+  sha256: a2c39153e425985b03427b1d1eab8a239b98272922ca49592c45c965b487ffbf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

Skip windows for now - it will try again once the R4.4 migrator visits the feedstock:

```suggestion
  skip: true  # [win]

```